umbraco-cms-webapp-redis-cache/azuredeploy.json

```diff
@@ -0,0 +1,525 @@
+         "defaultValue":"Standard"
+      },
+      "redisCacheCapacity":{
+         "type":"int",
+         "allowedValues":[
+            0,
+            1,
+            2,
+            3,
+            4,
+            5,
+            6
+         ],
+         "metadata":{
+            "description":"Redis Cache Capacity  , supported values from 0 to 6 represnting cache tier name C0 to C6 "
+         },
+         "defaultValue":0
+      },
+      "NonAdminDatabaseUsername":{
+         "type":"string",
+         "metadata":{
+            "description":"Non-admin Database User. Must be Unique"
+         }
+      },
+      "NonAdminDatabasePassword":{
+         "type":"securestring",
+         "metadata":{
+            "description":"Non-admin Database User password"
+         }
+      }
+   },
+   "resources":[
+      {
+         "apiVersion":"2014-04-01-preview",
+         "name":"[parameters('redisCacheName')]",
+         "type":"Microsoft.Cache/Redis",
+         "location":"[resourceGroup().location]",
+         "properties":{
+            "sku":{
+               "name":"[parameters('redisCacheSKU')]",
+               "family":"C",
+               "capacity":"[parameters('redisCacheCapacity')]"
+            },
+            "redisVersion":"2.8",
+            "enableNonSslPort":true
+         }
+      },
+      {
+         "apiVersion":"2.0",
+         "name":"[parameters('serverName')]",
+         "type":"Microsoft.Sql/servers",
+         "location":"[resourceGroup().location]",
+         "properties":{
+            "administratorLogin":"[parameters('administratorLogin')]",
+            "administratorLoginPassword":"[parameters('administratorLoginPassword')]"
+         },
+         "resources":[
+            {
+               "apiVersion":"2.0",
+               "name":"[parameters('databaseName')]",
+               "type":"databases",
+               "location":"[resourceGroup().location]",
+               "dependsOn":[
+                  "[concat('Microsoft.Sql/servers/', parameters('serverName'))]"
+               ],
+               "properties":{
+                  "edition":"[parameters('edition')]",
+                  "collation":"SQL_Latin1_General_CP1_CI_AS",
+                  "maxSizeBytes":"1073741824",
+                  "requestedServiceObjectiveId":"910b4fcb-8a29-4c3e-958f-f7ba794388b2"
+               }
+            },
+            {
+               "apiVersion":"2.0",
+               "name":"AllowAllWindowsAzureIps",
+               "type":"firewallrules",
+               "location":"[resourceGroup().location]",
+               "dependsOn":[
+                  "[concat('Microsoft.Sql/servers/', parameters('serverName'))]"
+               ],
+               "properties":{
+                  "endIpAddress":"0.0.0.0",
+                  "startIpAddress":"0.0.0.0"
+               }
+            }
+         ]
+      },
+      {
+         "apiVersion":"2014-06-01",
+         "type":"Microsoft.ClassicStorage/storageAccounts",
```

Comment:
Please use v2 storage. Only Ibiza subs are allowed to deploy classic storage accounts from ARM